.tav.yml

```diff
@@ -138,8 +138,11 @@ pg-new-node:
 # Latest mongodb-core release (v3.2.7) was released in 2019. We test a subset
 # of the full supported range.
 mongodb-core:
-  versions: '1.2.19 || 1.2.32 || 1.3.21 || 2.0.14 || 2.1.20 || 3.0.11 || 3.1.11 || 3.2.7' # latest minors subset of '>=1.2.19 <4'
+  versions: '1.2.19 || 1.2.32 || 1.3.21 || 2.0.14 || 2.1.20 || 3.0.11 || 3.1.11 || 3.2.7' 
```

Comment:
We talked about adding the caret to the last version. Do you intend to do that for this PR, or in a later one?

Reply:
and in some cases we talked about ending with something like `>3.2.7 <4`. Could we have one way of adding last versions? I'm planning to add it.

Reply:
Either form (caret or a range with `>... <...') sounds good to me.

Reply:
I'll go for caret :)